src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -114,100 +137,244 @@ object VCFOutput {
               // Call an alt if there is any variant evidence in this sample and no other alt allele has more evidence.
-              val topTwoAlleles = tumorEvidence.allelicDepths.filter(_._2 > 0).toSeq.sortBy(-1 * _._2).map(_._1).take(2).toSet
+              val topTwoAlleles =
+                tumorEvidence
+                  .allelicDepths
+                  .filter(_._2 > 0)
+                  .toSeq
+                  .sortBy(-1 * _._2)
+                  .map(_._1)
+                  .take(2)
+                  .toSet
+
               topTwoAlleles match {
-                case x if x == Set(allele.ref, allele.alt) => Seq(allele.ref, allele.alt)
-                case x if x == Set(allele.ref) || x == Set(allele.alt) => Seq(x.head, x.head)
-                case _ => Seq(allele.ref, allele.ref) // fall back on hom ref
+                case x if x == Set(allele.ref, allele.alt) =>
+                  Seq(allele.ref, allele.alt)
+
+                case x if x == Set(allele.ref) || x == Set(allele.alt) =>
+                  Seq(x.head, x.head)
+
+                case _ =>
+                  Seq(allele.ref, allele.ref) // fall back on hom ref
               }
-            }
-            case Parameters.SomaticGenotypePolicy.Trigger => {
+
+            case Parameters.SomaticGenotypePolicy.Trigger =>
               // Call an alt if this sample triggered a call.
               if (thisSampleTriggered)
                 Seq(allele.ref, allele.alt)
               else
                 Seq(allele.ref, allele.ref)
-            }
+
           }
-          genotypeBuilder.alleles(JavaConversions.seqAsJavaList(sampleGenotype.map(makeHtsjdkAllele _)))
-          genotypeBuilder.attribute("RL",
-            posteriors.map(pair => "[%s]=%.8g".format(mixtureToString(pair._1), pair._2)).mkString(","))
+
+          genotypeBuilder.alleles(
+            JavaConversions.seqAsJavaList(
+              sampleGenotype.map(
+                makeHtsjdkAllele _
+              )
+            )
+          )
+
+          genotypeBuilder.attribute("RL", mixturesToString(posteriors))
+
           genotypeBuilder.attribute("TRIGGERED", if (thisSampleTriggered) "YES" else "NO")
 
           genotypeBuilder
             .attribute("ADP", allelicDepthString(tumorEvidence.allelicDepths))
             .attribute("VAF", tumorEvidence.vaf)
-            .AD(Seq(allele.ref, allele.alt).map(allele => tumorEvidence.allelicDepths.getOrElse(allele, 0)).toArray)
+            .AD(
+              Seq(
+                allele.ref,
+                allele.alt
+              )
+              .map(allele => tumorEvidence.allelicDepths.getOrElse(allele, 0))
+              .toArray
+            )
             .DP(tumorEvidence.depth)
-        }
-        case (TissueType.Tumor, Analyte.RNA) => {
-          val tumorEvidence = samplesEvidence.allEvidences(input.index).asInstanceOf[TumorRNASingleSampleSingleAlleleEvidence]
+
+        case (TissueType.Tumor, Analyte.RNA) =>
+
```

Comment:
this 4-`case` `match` block is 150 lines long; I wanted to space out the division between `case`s here to make it easier to see.

Maybe later I'll factor it out to be less humongous, but I thought this was an improvement for now.